matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/sync/model/LocationDataEvent.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.session.sync.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.matrix.android.sdk.api.session.events.model.Content
+
+/**
+ * This is a simplified Event with just a content and a timestamp.
+ */
+@JsonClass(generateAdapter = true)
+data class LocationDataEvent(
+        @Json(name = "content") val content: Content?,
+        @Json(name = "timestamp") val timestamp: Long?
```

Comment:
I think @Danieloni1 changes it to `origin_server_ts` ?

Reply:
@danpe @valerianglobekeeper 
True! Just merged it to staging :)